inc/itilfollowup.class.php

```diff
@@ -330,18 +336,16 @@ function post_addItem() {
          $update['id']     = $parentitem->fields['id'];
 
          // don't notify on ITILObject - update event
-         $update['_disablenotif'] = true;
+         if (!$donotif
+             || ($this->input['_status'] != CommonITILObject::SOLVED
+                 && $this->input['_status'] != CommonITILObject::CLOSED)) {
```

Comment:
Changes and Problems have multiple statuses that matches the `solved` state.
```suggestion
             || (!in_array($this->input['_status'], $this->getSolvedStatusArray())
                 && !in_array($this->input['_status'], $this->getClosedStatusArray()))) {
```